include/SFML/Config.hpp

```diff
@@ -123,7 +123,7 @@
 ////////////////////////////////////////////////////////////
 #if !defined(NDEBUG)
 
-    #define SFML_DEBUG
+#define SFML_DEBUG
 
 #endif
```

Comment:
Since this is just a list of marcos, it might make sense to preserve the formatting here, otherwise it becomes harder to read

Reply:
Maybe finally time to move to `#pragma once` 😛

Reply:
I ran a quick test and can confirm that with `#pragma once` the preprocessor indentation behaves as expected and desired. With header guards, some files result in code like 
```cpp
#ifndef SFML_TIME_HPP
    #define SFML_TIME_HPP

    ////////////////////////////////////////////////////////////
    // Headers
    ////////////////////////////////////////////////////////////
    #include <SFML/System/Export.hpp>
```
but other files get their preprocessor directives indented as desired. So to get the preprocessor indentation without other undesirable side effects, we need to use `#pragma once`. Personally I don't mind left-indenting all preprocessor directives but I don't feel strongly either way.

Reply:
I think it's generally okay to not indent preprocessor directives, just thought maybe here disabling formatting might be useful. Maybe also in the Export.hpp